jnotebook-core/src/main/java/tech/catheu/jnotebook/jshell/ShellProvider.java

```diff
@@ -110,13 +107,21 @@ private String computeMavenClasspath() throws IOException, InterruptedException
       LOG.warn("Maven wrapper not found. Trying to use `mvn` directly.");
       mavenExecutable = "mvn";
     }
-    final String cmd = mavenExecutable + MAVEN_DEPENDENCY_COMMAND;
+    final String classpathCommand = IS_OS_WINDOWS ? MAVEN_DEPENDENCY_COMMAND_WINDOWS : MAVEN_DEPENDENCY_COMMAND;
+    final String cmd = mavenExecutable + classpathCommand;
     final Runtime run = Runtime.getRuntime();
     final Process pr = run.exec(cmd);
     pr.waitFor();
     final BufferedReader reader =
             new BufferedReader(new InputStreamReader(pr.getInputStream()));
     final List<String> classpaths = reader.lines().toList();
+    // mvn (-q) does not suppress errors, but all error line start with "[ERROR]"
+    // there's cases where we get some classpaths, but some module fails, we can load the successful ones.
+    final List<String> errors = classpaths.stream().filter(s -> s.startsWith("[ERROR]")).toList();
```

Comment:
let's use the exit code of the process directly, it'll be more solid.

You can check
```
pr.exitValue()
```
just after `pr.waitFor()`

If the exit value is not zero, then the process failed and you can throw the RunTimeException.
Also, let's return the full output in this case.

Reply:
and maybe add the input command. Such that the user can debug easily.